sdk/formrecognizer/azure-ai-formrecognizer/tests/test_repr.py

```diff
@@ -439,6 +439,53 @@ def doc_type_info():
     assert repr(model) == model_repr
     return model, model_repr
 
+
+@pytest.fixture
+def blob_file_list_source():
+    model = _models.BlobFileListSource(
+            container_url="https://test.blob.core.windows.net/blob-sas-url",
+            file_list="filelist.jsonl",
+    )
+    model_repr = f"BlobFileListSource(container_url={'https://test.blob.core.windows.net/blob-sas-url'}, file_list={'filelist.jsonl'})"
```

Comment:
do we need the curly brackets?

edit: few instances where I'm not sure if the string needs to be wrapped in curly brackets or if it's only passing like that...

Reply:
Copilot helped me write this! But for an instance where I didnt include the curly brackets it did fail, because then the string in the repr doesnt look the way it would through the actual method. If we wanted to remove the curly brackets then we would just pass the value in like `file_list=filelist.jsonl`.

Reply:
I like the curly brackets because it kind of mimics what is happening with repr when we pass in a string or other value

Reply:
I didn't think the repr included the `{}`, that was just there for the format. Let me check

Reply:
Ah yeah you're right, yeah I can remove the curly brackets